web_frontend/config/urls.py

```diff
@@ -10,4 +10,5 @@
     url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
     url(r'^api/', include('osmaxx.excerptexport.rest_api.urls', namespace='excerptexport_api')),
     url(r'^job_progress/', include('osmaxx.job_progress.urls', namespace='job_progress')),
+    url(r'^pages/', include('osmaxx.core.urls', namespace='pages')),
```

Comment:
Let's keep it as-is-in-this-PR for now and discuss this again when the (dynamic) downloads page gets revised anyway (due to focusing on excerpts rather than orders).